packages/react-events/src/Press.js

```diff
@@ -125,17 +126,19 @@ const rootEventTypes = [
   'scroll',
   'pointercancel',
 ];
+const baseRootEventTypes = [
+  // Used as a 'cancel' signal for mouse interactions
```

Comment:
Either this comment should exist above `dragstart` below, or `dragstart` should be removed if it's not used anymore.